modules/agar/package.json

```diff
@@ -23,7 +23,7 @@
     "test"
   ],
   "author": "Ephox Corporation DBA Tiny Technologies, Inc",
-  "license": "MIT",
+  "license": "GPL-2.0-or-later",
```

Comment:
We will need new major versions for every library

- [ ] bump the majors (should be an easy lerna command)

Reply:
Bumped in the version.txt file that is how we normally do it unless that has now changed. Since doing a `lerna version` will treat what is there now as the release and start tagging things.